react/utils/components/index.ts

```diff
@@ -87,6 +87,7 @@ export const getExtension = (
     implements: extensionImplements,
     props: props || {},
     shouldRender,
+    title: '',
```

Comment:
I believe the title should come from `extensions[editTreePath!] || {}`, with a fallback to `''`. The way it is, `title` will always be `''`.